components/script/dom/document.rs

```diff
@@ -936,6 +936,10 @@ impl Document {
         point_in_node: Option<Point2D<f32>>,
         pressed_mouse_buttons: u16,
     ) {
+        if self.window.Closed() {
+            // If the BC is being discared, we don't want to run any JS.
+            return;
```

Comment:
If this is generally true, should we include it in codegen?

Reply:
I'm not familiar with codegen, how would that work?

Reply:
Note to self: fix the typo in "discarded"